File: challenges/2-idade-em-dias/java/gdiasb/IdadeEmDias.java
import java.util.Scanner;

public class IdadeEmDias {

  public static void main(String[] args) {

    Scanner scanner = new Scanner(System.in);

    System.out.println("Digite sua idade em dias: ");
    int idadeDias = scanner.nextInt();
    scanner.close();

    int diasDoAno = 365;
    int diasDoMes = 30;

    int quantidadeAnos = idadeDias / diasDoAno;
    int quantidadeMeses = (idadeDias % diasDoAno) / diasDoMes;
    int quantidadeDias = ((idadeDias % diasDoAno) % diasDoMes);

    System.out.println(quantidadeAnos + " ano(s), " + quantidadeMeses + " mes(es) e " + quantidadeDias + " dia(s)");

  }
}

File: challenges/2-idade-em-dias/python/trustindestiny/idade-em-dias.py
idade = int(input('Digite sua idade em dias: '))
calc_anos = idade // 365
calc_meses = (idade % 365) // 30
calc_dias = (idade % 365) % 30

print(f'{calc_anos} Ano(s)')
print(f'{calc_meses} Mes(es)')
print(f'{calc_dias} Dia(s)')
